### ISSUE TITLE
Styling not working properly

### ISSUE THREAD

**Issue by tomasbaran** [DavBfr/dart_pdf#342]
I also have problems with styling. These don't work:

## Styling issue 1

`style: TextStyle(color: pw.Colors.black,),`
### Error
The name 'Colors' is being referenced through the prefix 'pw', but it isn't defined in any of the libraries imported using that prefix.
Try correcting the prefix or importing the library that defines 'Colors'.

## Styling issue 2
`style: tsQRcode.copyWith(fontWeight: FontWeight.w500),` 
```
TextStyle tsQRcode = GoogleFonts.nunitoSans(
  textStyle: TextStyle(
    color: Colors.black,
    letterSpacing: 1,
    fontSize: 24,
    fontWeight: FontWeight.w900,
  ),
);

```
### Error
The argument type 'TextStyle (where TextStyle is defined in /Users/tomas/Development/flutter/packages/flutter/lib/src/painting/text_style.dart)' can't be assigned to the parameter type 'TextStyle (where TextStyle is defined in /Users/tomas/.pub-cache/hosted/pub.dartlang.org/pdf-1.8.1/lib/widgets/text_style.dart)'.

## Styling issue 3

letter spacing is being ignored:

```
pw.Text(
                'text',
                style: pw.TextStyle(
                  letterSpacing: 8,
                ),
              ),
```

**Comment by DavBfr** [comment]
You can't mix Flutter Widgets and objects with Pdf Widgets. It's not the same library. the Pdf library can work without flutter and reimplements a part of the flutter widgets.

The right black color is `PdfColors.black`. GoogleFonts can be used only within Flutter, not Pdf. A GitHub issue is pending on their repository.

**Comment by tomasbaran** [comment]
Thanks and the last one that you didn't mention is letterSpacing. how can we achieve that?

**Comment by DavBfr** [comment]
`letterSpacing` is not yet implemented in the backend. You can create a feature request for it.

**Comment by DavBfr** [comment]
Fixed in version 1.9.0
